packages/block-library/src/loginout/edit.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * WordPress dependencies
+ */
+import { PanelBody, ToggleControl, Disabled } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import ServerSideRender from '@wordpress/server-side-render';
+
+export default function LoginOutEdit( { attributes, setAttributes } ) {
+	const { displayLoginAsForm, redirectToCurrent } = attributes;
+
+	return (
+		<>
+			<InspectorControls>
+				<PanelBody title={ __( 'Login/out settings' ) }>
+					<ToggleControl
+						label={ __( 'Display login as form' ) }
+						checked={ displayLoginAsForm }
+						onChange={ () =>
+							setAttributes( {
+								displayLoginAsForm: ! displayLoginAsForm,
+							} )
+						}
+					/>
+					<ToggleControl
+						label={ __( 'Redirect to current URL' ) }
+						checked={ redirectToCurrent }
+						onChange={ () =>
+							setAttributes( {
+								redirectToCurrent: ! redirectToCurrent,
+							} )
+						}
+					/>
+				</PanelBody>
+			</InspectorControls>
+			<div { ...useBlockProps() }>
+				<Disabled>
+					<ServerSideRender
```

Comment:
Why using `ServerSideRender` here? When editing the user will always be logged in. Also with server side by changing props there is that quick reload state that doesn't change anything. A simple placeholder could be better I think.

Reply:
Good point. Fixed in https://github.com/WordPress/gutenberg/pull/29766/commits/4c3a1fa853a5fb4b670187934f58b7b1118c99f9 👍